devito/compiler.py

```diff
@@ -225,7 +224,6 @@ def make(self, loc, args):
         logfile = path.join(self.get_jit_dir(), "%s.log" % hash_key)
         errfile = path.join(self.get_jit_dir(), "%s.err" % hash_key)
 
-        tic = time()
```

Comment:
why? Actually useful I find to know how ling it takes to compile

Reply:
`make` is never used except for yask

now timings all go via `timed_pass`

Reply:
ok